### DISCUSSION TITLE
The selected topic on feed gone after refresh

### DISCUSSION THREAD

**Discussion by wirayuda299** [community/community#54858] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Everytime i select a topic fior my Feed it's gone on refresh

**Comment by iamreiyn** [comment]
Hi @wirayuda299 

If you are using GitHub's web UI and find that your selected topic filter on your feed is gone after refreshing, it may be due to your browser cache.

To resolve this issue, you can try clearing your browser's cache and refreshing the page or pressing Ctrl + Shift + R on the page. Also, make sure to click on the "Apply changes" button after selecting your filters.

If the issue persists, you can try logging out and then logging back into your GitHub account. Thank you.

**Comment by hamidrezamd1380** [comment]
https://realit.ir/
